core/src/com/punchbrain/awake/behavior/HoundBehavior.java

```diff
@@ -68,6 +90,25 @@ public void update(final GameObject hound, final float delta) {
         lightingManager.setAmbientLight(Colors.darken(com.punchbrain.awake.Colors.BACKGROUND, ratio));
     }
 
+    @Override
+    public void update(GameObject hound, GameObject object, float delta){
+        if(CIRCUIT_LAMP.isTypeOf(object)){
+            Circuit circuit = circuitModelMap.getFromLamp(object);
+            if(circuit.getState() == Circuit.State.ON){
+                double xSep = hound.getLeft() - object.getLeft();
+                double ySep = hound.getTop() -  object.getTop();
+                double rSep = Math.pow(Math.pow(xSep, 2) + Math.pow(ySep, 2), 0.5);
```

Comment:
you actually don't need that. [Checkout the RayHandler class](https://github.com/libgdx/box2dlights/blob/master/src/box2dLight/RayHandler.java#L401) available through the LightingManager. I'd prefer this since we then can use different types of lights (like cone light for example)